### PR TITLE
[HTML5] show audio modal and toast notification when breakout room ends

### PR DESCRIPTION
**Pull request by Tainan404**
- Show toast notification when breakout room notifying the user to rejoin audio.
- Show audio modal when breakout room ends
- Wait for PR #4687 that fixes the leaves audio for breakout room.

### CODE REVIEW

File: bigbluebutton-html5/private/locales/en.json [hunk 1]
```diff
@@ -255,6 +255,7 @@
     "app.error.403": "Forbidden",
     "app.error.leaveLabel": "Log in again",
     "app.guest.waiting": "Waiting for approval to join",
+    "app.toast.BreakoutRoomEnded": "The breakout room ended. Please rejoin in the audio.",
```
Comment (oswaldoacauan) [inline bigbluebutton-html5/private/locales/en.json]: `app.toast.breakoutRoomEnded`

**Comment by OZhurbenko** [comment]
I tested it, and it didn't work for me.

Ended a breakout room -> the html5 breakout tab closed -> notification showed up as expected, but the "Join Audio" button was disabled.

And the only way to get out of that state is to refresh the page.

**Comment by antobinary** [comment]
Please wait for the merge of PR #4687 which fixes the leaves audio for breakout room.

**Comment by antobinary** [comment]
@Tainan404 #4687 was merged. Is this ready for testing?

**Comment by Tainan404** [comment]
@antobinary PR this ready for test.

**Comment by OZhurbenko** [comment]
String id is shown instead of the message in the toast notification:
toast_notification_breakout_ended

**Pull request merged**